pkgs/applications/version-management/git-lfs/default.nix

```
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "git-lfs";
-  version = "3.4.1";
+  version = "cd04b35";
```

For non-tagged releases, please use `unstable-YY-MM-DD`. That way it's order-able.

---

Also please add a comment as to why we are not using a tagged release
```suggestion
  # Inclues git compatibility fixes which are in master, but no official release
  version = "cd04b35";
```